<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientManager</class>
 <widget class="QWidget" name="PatientManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="waitTreeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>순번</string>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>100</red>
            <green>100</green>
            <blue>100</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>환자ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>100</red>
            <green>100</green>
            <blue>100</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>환자명</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>100</red>
            <green>100</green>
            <blue>100</blue>
           </color>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>삭제</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="patientReadyButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>환자 준비</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="infoTableWidget">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>16777215</height>
          </size>
         </property>
         <row>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>촬영 타입</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>이름</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>나이</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>성별</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <column>
          <property name="text">
           <string>내용</string>
          </property>
          <property name="toolTip">
           <string/>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finishButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>촬영 완료</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
